Update index.html

badlard opened the pull request:
Updates to reflect the primary "service designer" persona

Commit:
Update index.html

Updates to reflect the primary "service designer" persona

justinvdm commented on index.html:
```diff
@@ -22,46 +22,40 @@
+                <p>Sarima seeks to <strong>understand the needs </strong> of every stakeholder and <strong>design the system</strong> so that it meets stakeholder needs.</p>
 
-                <p>For in-country deploys he wants to <strong>choose the Vumi components that he needs</strong> and just use those. <em>:moonshot:</em></p>
+                <p>Sarima needs to <strong>conduct tests with users</strong> using either paper or clickable prototypes on a mobile device. She needs the ability to <strong>easily update her prototype</strong> in the field and in co-design sessions. She mainly uses her laptop but sees value in making updates via her mobile. She often <strong>exports data and crunches it in Excel</strong> to get t ...
```
> `Sarima needs to <strong>conduct tests with users</strong> using either paper or clickable prototypes on a mobile device`

If I'm remembering correctly from when we chatted to Jacqui, I think paper prototypes are one possible case for when testing on a phone isn't possible, but most of the time (or at least the ideal case) is to test it as an actual USSD service (if the plan is for it to be over USSD) rather than in a web/native app's UI on a mobile.

> `She mainly uses her laptop but sees value in making updates via her mobile`

Wondering if it would be helpful to mention the use case for the mobile (e.g. when co-designing with client).

> `She often <strong>exports data and crunches it in Excel</strong> to get the reports she needs`

Two things:
- From what I understand from our discussion with Jacqui, I think there are important analysis tasks, for e.g. seeing the results of a change they have made to the system. Is that included in the reports mentioned here? Wondering if we should mention it somehow?
- Wondering if we need to mention cases where the service designer would want to see something in a UI/dashboard/graph rather than export and post-process? Maybe things like analysing which screens have high drop-off rates?

justinvdm commented:
Wondering if we should remove Olana the PM?

justinvdm commented:
Looks good :) Left a few comments.

Commit:
Update index.html

Made updates to 1) reflect her preference for prototyping on the device 2) better reflect the type of analysis she needs to conduct. I think we should leave the PM persona in for now, though it needs to be updated as well in a future sprint task.

justinvdm commented on index.html:
```diff
@@ -22,46 +22,42 @@
-                <p>For in-country deploys he wants to <strong>choose the Vumi components that he needs</strong> and just use those. <em>:moonshot:</em></p>
+                <p>Sarima needs to <strong>conduct tests with users</strong> over USSD/SMS/chat on a mobile device. She needs the ability to <strong>easily update her prototype</strong> in the field and in co-design sessions. Making these updates via her mobile phone would be useful. 
+                
+                <p>Sarima needs to see <strong>how users are moving through a flow</strong> and which screens are experiencing high drop-off rates. She also needs to analyze the effect of changes she has made to the system flow. 
```
Exporting and post-processing in Excel (for analysis or to report to client) seems important, would it be worth bringing it back?

justinvdm commented:
Changes look good, left one more minor comment.

hodgestar commented:
Something in this reminded me of the idea of simulating interactions with a
collection. Is this an idea we want to retain in Numi?

hodgestar commented:
Second idea -- have we thought about printing paper prototypes from Numi
collection definitions?

Commit:
Update index.html

Small addition re: exporting data to Excel.

badlard commented:
@justinvdm made a small addition re: exporting to Excel.

justinvdm commented:
:+1: (sorry, forgot about this PR)
